ext/InverseFunctionsUnitfulExt.jl

```diff
@@ -0,0 +1,8 @@
+module InverseFunctionsUnitfulExt
+using Unitful
+import InverseFunctions: inverse
+
+# `true` plays the role of 1, but doesn't promote unnecessary
+inverse(f::Base.Fix1{typeof(ustrip)}) = Base.Fix1(*, true*f.x)
```

Comment:
```suggestion
inverse(f::Base.Fix1{typeof(ustrip), <:Unitful.Units}) = Base.Fix1(*, true*f.x)
```
I would restrict the type of the first `ustrip` argument, because there is currently a discussion about adding another two-argument `ustrip` method where the first argument isn’t a unit (`ustrip(reinterpret, x)`, cf. https://github.com/PainterQubits/Unitful.jl/issues/630#issuecomment-1482765400)